bundles/org.openhab.binding.solarmax/README.md

```diff
@@ -0,0 +1,58 @@
+# SolarMax Binding
+
+This binding supports SolarMax PV inverters.
+
+## Supported Things
+
+This binding only has a single `inverter` thing that can be added manually.
+The SolarMax MT Series is supported (tested with 8MT2 devices).
+
+## Discovery
+
+Auto-discovery is currently not available.
+
+## Thing Configuration
+
+Each inverter requires the following configuration parameters:
+
+| parameter       | required | default | description                                                          |
+| --------------- | -------- | ------- | -------------------------------------------------------------------- |
+| host            | yes      |         | hostname or IP address of the inverter                               |
+| port            | no       | 12345   | Port number to connect to. This should be `12345` for most inverters |
+| refreshInterval | no       | 15      | Interval (in seconds) to refresh the channel values.                 |
+
+
+## Channels
+
+| channel                  | type     | description                                            |
+| ------------------------ | -------- | ------------------------------------------------------ |
+| lastUpdated              | DateTime | Time when data was last read from the device           |
+| softwareVersion          | Point    | Software Version installed on the SolarMax device      |
+| buildNumber              | Point    | Firmware Build Number installed on the SolarMax device |
```

Comment:
These seem rather properties than Channels.

Reply:
I currently have no time to read into what would need to be changed, make the change and test it. Since it currently works as required, I hope it's okay if we put it in the backlog for next time.

Reply:
Changing these to properties would be a breaking change, once released. If you don't have the time to take a look into this, you could remove them for now and add them in a future release.

Reply:
So I had a crack at this and it seems to be working now returning these values as properties.